services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/IslMovingSpec.groovy

```diff
@@ -30,7 +30,9 @@ class IslMovingSpec extends BaseSpecification {
         assert isl
 
         and: "A non-connected a-switch link"
-        def notConnectedIsl = topology.notConnectedIsls.first()
+        def notConnectedIsl = topology.notConnectedIsls.find {
+            it.srcSwitch != isl.srcSwitch && it.srcSwitch != isl.dstSwitch 
```

Comment:
if possible, please change this line to
!(it.srcSwith in [isl.srcSwitch, isl.dstSwitch])
just to avoid possible confusion of automated static code analysis